php/docker/templates/nginx-conf.d/90-metrics.conf

```
@@ -0,0 +1,87 @@
+{{- $metricsEnabled := default "true" .Env.STACK_METRICS_ENABLED }}
+
+{{- if isTrue $metricsEnabled }}
+
+lua_shared_dict prometheus_metrics 10M;
+
+{{- $siteName := default "" .Env.STACK_SITE_NAME }}
+
+init_worker_by_lua_block {
+    prometheus = require("prometheus").init("prometheus_metrics")
+    metric_requests = prometheus:counter(
+        "nginx_http_requests_total",
+        "Number of HTTP requests",
+        {"host", "site_name", "status", "cache_status"}
+    )
+    metric_request_time = prometheus:histogram(
+        "nginx_http_request_duration_seconds",
+        "HTTP request request duration",
+        {"host", "site_name"}
+    )
+    metric_connections = prometheus:gauge(
+        "nginx_http_connections",
+        "Number of HTTP connections",
+        {"host", "site_name", "state"}
+    )
+}
+
+log_by_lua_block {
+    metric_requests:inc(
+        1, {
+            ngx.var.server_name,
+            {{ $siteName }}
```

I think you need to quote this because otherwise, you can end up with an empty line.  The default value of`$siteName`  is an empty string, not two quotes. I'm not very sure :smile: 

```suggestion
            "{{ $siteName }}"
```

---

I fixed this by using `| qoute`